features-json/css-focus-visible.json

```diff
@@ -177,8 +185,8 @@
       "64":"n",
       "65":"n",
       "66":"n",
-      "67":"n",
-      "68":"n"
+      "67":"p d #2",
+      "68":"p d #2"
```

Comment:
Forgive my drive-by reviewing: Do you mean `"a d #2"` here? `p` means "No support, but has polyfill", whereas `a` means "almost supported" and corresponds to "partial support" in the caniuse.com UI.

Reply:
oh, yes I think that's what I want then :) I'll update. Thanks!